src/main/java/core/basesyntax/service/FruitList.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public interface FruitList {
+    List<FruitTransaction> getFruitList(List<FruitTransaction> list);
```

Comment:
The method name 'getFruitList' is not descriptive of the action it is supposed to perform. A name like 'processTransactions' or 'applyTransactions' might be more indicative of its functionality.